components/generic/RelatedCollections.vue

```diff
@@ -29,50 +30,59 @@
     },
 
     props: {
-      query: {
+      title: {
         type: String,
         default: ''
+      },
+      relatedCollections: {
+        type: Array,
+        default: () => []
       }
     },
 
-    fetch() {
-      this.getSearchSuggestions(this.query);
-    },
-
-    data() {
-      return {
-        relatedCollections: []
-      };
-    },
-
     computed: {
       ...mapGetters({
         apiConfig: 'apis/config'
       })
     },
 
-    watch: {
-      query: '$fetch'
-    },
-
     methods: {
-      async getSearchSuggestions(query) {
-        this.relatedCollections = query === '' ? [] : await getEntitySuggestions(query, {
-          language: this.$i18n.locale,
-          rows: 4
-        });
-      },
+      linkGen(item) {
+        let id = '';
+        let name = '';
+
+        if (item.id) {
+          id = item.id;
+          name = item.prefLabel[this.$i18n.locale];
+        } else {
+          id = item.identifier;
+          name = item.name;
+        }
 
-      suggestionLinkGen(item) {
-        let id = item.id;
-        let name = item.prefLabel[this.$i18n.locale];
         const uriMatch = id.match(`^${this.apiConfig.data.origin}/([^/]+)(/base)?/(.+)$`);
         return this.$path({
           name: 'collections-type-all', params: {
             type: getEntityTypeHumanReadable(uriMatch[1]),
             pathMatch: getEntitySlug(id, name)
           }
         });
+      },
+      collectionTitle(item) {
+        if (item.prefLabel) {
+          return item.prefLabel[this.$i18n.locale];
+        } else {
+          return item.name;
+        }
```

Comment:
Perhaps
`return item.prefLabel ? item.prefLabel[this.$i18n.locale] : item.name`

Reply:
yes thats better